### PR TITLE
Wqp 1267

### PR DESCRIPTION
**Pull request by dsteinich**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/java/gov/usgs/cida/wqp/mapping/ResultColumn.java [hunk 1]
```diff
@@ -8,7 +8,11 @@
 	public static final String KEY_ANALYTICAL_METHOD = "ANALYTICAL_METHOD";
 	public static final String KEY_RES_DATA_LOGGER_LINE = "RES_DATA_LOGGER_LINE";
 	public static final String KEY_RESULT_DETECTION_CONDITION_TX = "RESULT_DETECTION_CONDITION_TX";
+
+	//Note that "SPECIATION" is correct and "SPECIFICATION" is a typo.
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/wqp/mapping/ResultColumn.java]: Thanks for the clarification, I wouldn't have known that.

Comment (ssoper-usgs) [reply src/main/java/gov/usgs/cida/wqp/mapping/ResultColumn.java]: Should METHOD_SPECIFICATION_NAME be removed at some point if it's a typo?

File: src/test/java/gov/usgs/cida/wqp/dao/streaming/result/ResultPhysChemStreamingIT.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+		stateTest(nameSpace);
+		subjectTaxonomicNameTest(nameSpace);
+		withinTest(nameSpace);
+		zipTest(nameSpace);
+		multipleParameterResultTest(nameSpace);
+		multipleParameterResultStationSumTest(nameSpace);
+	}
+
+	@Override
+	protected void assertSiteUrlBase(Map<String, Object> row) {
```
Comment (ssoper-usgs) [inline src/test/java/gov/usgs/cida/wqp/dao/streaming/result/ResultPhysChemStreamingIT.java]: Is this method abstract up in the BaseResultStreamingTest?

File: src/test/java/gov/usgs/cida/wqp/webservice/result/NarrowResultControllerIT.java [hunk 3]
```diff
@@ -41,115 +25,25 @@
...
+		getAsCsvTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsCsvZipTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsTsvTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsTsvZipTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsXlsxTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsXlsxZipTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsXmlTest(ENDPOINT, PROFILE, POSTABLE);
+		getAsXmlZipTest(ENDPOINT, PROFILE, POSTABLE);
+		getAllParametersTest(ENDPOINT, PROFILE, POSTABLE);
+		postGetCountTest(PROFILE);
```
Comment (ssoper-usgs) [inline src/test/java/gov/usgs/cida/wqp/webservice/result/NarrowResultControllerIT.java]: Thanks for consolidating some of these tests.

File: src/main/java/gov/usgs/cida/wqp/mapping/ResultColumn.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/cida/wqp/mapping/ResultColumn.java]: Up to product owner as to when we could depreciate it. It cannot be removed until after community has a chance to update...

File: src/test/java/gov/usgs/cida/wqp/dao/streaming/result/ResultPhysChemStreamingIT.java (hunk 2 above)
Comment (dsteinich) [reply src/test/java/gov/usgs/cida/wqp/dao/streaming/result/ResultPhysChemStreamingIT.java]: It is actually an abstract in FilteredDaoTest (and should be doing something in the class...)

**Pull request merged**